docs/userguide/storagedriver/device-mapper-driver.md

```diff
@@ -304,6 +304,35 @@ options were created in the previous steps.
  The last two lines also confirm that the correct devices are being used for
 the `Data file` and the `Metadata file`.
 
+### Extend devicemapper thin-pool
+
+#### compile device_tool
+
+You have to compile contrib/docker-device-tool/device_tool.go [reference](https://github.com/docker/docker/blob/master/contrib/docker-device-tool/README.md)
+
+#### loop-lvm mode
+
```

Comment:
I'm following the above document `Docker hosts running the devicemapper storage driver default to a configuration mode known as loop-lvm`